lib/nodeshift-config.js

```diff
@@ -29,10 +29,17 @@ async function setup (options = {}) {
   options.nodeshiftDirectory = '.nodeshift';
   options.projectLocation = options.projectLocation || process.cwd();
 
+  if (options.knative) {
+    logger.warning('Using the EXPERMINTAL FLAG knative flag.');
+  }
+
   logger.info('loading configuration');
   const projectPackage = JSON.parse(await readFile(`${options.projectLocation}/package.json`, { encoding: 'utf8' }));
   // If there is a configLocation string, pass it in
-  const restClient = await openshiftRestClient({ config: options.configLocation });
+  const restClient = await openshiftRestClient({ config: options.configLocation, loadSpecFromCluster: options.knative });
```

Comment:
Is `loadSpecFromCluster` a temporary thing? Would it ever be beneficial to just always load it from the cluster?

Reply:
At some point in the future i think i would like make this the default.  It would require a major version bump though.  

So holding off for now